packages/openneuro-app/src/scripts/datalad/mutations/add-metadata.jsx

```diff
@@ -82,7 +82,7 @@ const runValidations = values =>
     .map(validation => {
       const relevantValues = validation.fields.map(key => values[key])
       // TODO - This doesn't seem necessary?
-      // @ts-expect-error
+      // @ts-ignore
```

Comment:
Curious what requires changing these to @ts-ignore?

Reply:
When I ran `yarn tsc -b`, I kept getting an `Unused '@ts-expect-error' directive` message yesterday. When I took out the line, then the compiler failed. It seems to be working now, so I'll be change it back.